apps/approval/models.py

```diff
@@ -30,6 +32,22 @@ class Approval(models.Model):
     approved = models.BooleanField(_("godkjent"), default=False, editable=False)
     message = models.TextField(_("melding"))
 
+    def alert_user():
+        if approved:
+            header_message = "Ditt medlemskap i Online er godkjent"
+        else:
+            header_message = "Ditt medlemskap i Online er ikke godkjent"
```

Comment:
I think we can keep the header free of "spoilers", so that you have to open the email to read what it contains.